dashboard/models.py

```diff
@@ -171,8 +205,9 @@ class Course(models.Model):
     GRADING_CHOICES = [('Percent', 'Percent'), ('Point', 'Point'), ]
     show_grade_type = models.CharField(verbose_name="Show Grade Type", max_length=255,
                                          choices=GRADING_CHOICES, default='Percent')
+    data_last_updated = models.DateTimeField(verbose_name="Last time data last updated for this course", null=True, blank=True)
```

Comment:
```suggestion
    data_last_updated = models.DateTimeField(verbose_name="Time data for this course was last updated", null=True, blank=True)
```

Reply:
Would have to re-generate the migration for that. Not a huge deal, but it was weird to me "last" was repeated.

Reply:
Yeah, I can fix this in the migration too.